app/policies/organizations/invitation_policy.rb

```diff
@@ -1,5 +1,6 @@
 class Organizations::InvitationPolicy < ApplicationPolicy
   pre_check :verify_organization!
+  pre_check :verify_active_staff!
```

Comment:
I figured some of these admin only actions should still have `verify_active_staff!` incase an admin's staff account was deactivated for some reason.